collection.go

```diff
@@ -213,29 +213,17 @@ func (c *Collection) Name() string {
 	return c.name
 }
 
-// SetData replaces the data map with something else
-// Useful for persistent collections
-func (c *Collection) SetData(data map[string][]string) {
```

Comment:
We cannot remove this function, it's an api breaking change, we can do it for v3

Reply:
Shall we mark this as deprecated?